delete_non_appearing_companions.sql

```diff
@@ -0,0 +1,14 @@
+WITH NonAppearingCompanions(id)
```

Comment:
would this do the job ?
```suggestion
DELETE comp
FROM companion comp 
LEFT JOIN EpisodeCompanion AS ec ON ec.CompanionID = c.CompanionID
WHERE ec.EpisodeCompanionId is null```